src/providers/data_provider_opts.c

```diff
@@ -249,7 +249,7 @@ static int dp_copy_options_ex(TALLOC_CTX *memctx,
             }
             if (ret != EOK) {
                 DEBUG(SSSDBG_CRIT_FAILURE,
-                      "Failed to retrieve value for option (%s)\n",
+                      "Failed to copy value for option (%s)\n",
```

Comment:
Here we have yet different tag for string - round brackets. I know it is the same as before change but maybe we should unify this?

Reply:
Maybe. But not within this PR.